lib/builder-webpack4/src/preview/iframe-webpack.config.ts

```diff
@@ -124,7 +124,7 @@ export default async ({
     // stats: 'errors-only',
     output: {
       path: path.resolve(process.cwd(), outputDir),
-      filename: '[name].[hash].bundle.js',
+      filename: isProd ? '[name].[contenthash:8].bundle.js' : '[name].bundle.js',
```

Comment:
is contenthash applicable to webpack4?

Reply:
Sure https://v4.webpack.js.org/configuration/output/